content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -32,37 +32,45 @@ different approach.
 
 ## The Kubernetes network model
 
-Every `Pod` gets its own IP address. This means you do not need to explicitly
-create links between `Pods` and you almost never need to deal with mapping
-container ports to host ports.  This creates a clean, backwards-compatible
-model where `Pods` can be treated much like VMs or physical hosts from the
-perspectives of port allocation, naming, service discovery, load balancing,
-application configuration, and migration.
+Every `Pod` gets its own IP address, maximum one per IP family. This means
+you do not need to explicitly create links between `Pods` and you almost
+never need to deal with mapping container ports to host ports.  This creates
+a clean, backwards-compatible model where `Pods` can be treated much like
+VMs or physical hosts from the perspectives of port allocation, naming,
+service discovery, load balancing, application configuration, and migration.
+
+Kubernetes IP addresses exist at the `Pod` scope - containers within a `Pod`
+share their network namespaces - including their IP address. This means that
+containers within a `Pod` can all reach each other's ports on `localhost`.
+This also means that containers within a `Pod` must coordinate port usage,
+but this is no different from processes in a VM.  This is called the
+"IP-per-pod" model.
+
+In every cluster, there exists an abstract pod-network to which pods
+are connected by default, unless explicitly configured to use the
+host-network (on platforms that support it). Even if a host has
+multiple IPs, host-network pods only have one Kubernetes IP address at
```

Comment:
Perhaps a cleaner wording:

> Even if a host has multiple IPs, the Pod's Status.PodIPs field will only ever contain one IP per address family.

Reply:
there are not references to the fields before, that is why omitted that approach ... let's see what the other say, 

What  should be clear is that `Kubernetes IPs = Pod.Status.PodIPs`

Reply:
If we do mention the API, let's capitalize fields the way they appear in the API (`status.podIPs`).

Reply:
I personally prefer making explicit references to the API. As we all know, if it's in etcd, it must be true :-).

More seriously, the API is how people interact with the cluster. In other words, what does it mean for a pod to "have" an IP?